docs/standards/smart-contracts/lsp2-json-schema-utils.md

```diff
@@ -0,0 +1,339 @@
+---
+title: LSP2JSONSchemaUtils
+sidebar_position: 12
+---
+
+# LSP2JSONSchemaUtils
+
+:::info
+
+[`LSP2Utils.sol`](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP2ERC725YJSONSchema/LSP2Utils.sol)
+
+:::
+
+This library should be used to generate ERC725Y keys described in the JSON Schema.
+
+## Functions
+
+### generateSingletonKey
+
+```solidity
+function generateSingletonKey(
+    string memory keyName
+) internal pure returns (bytes32);
+```
+
+Generates a data key of `keyType` Singleton.
+
+#### Parameters:
+
+| Name      | Type   | Description                                          |
+| :-------- | :----- | :--------------------------------------------------- |
+| `keyName` | string | The string to hash to generate a Singleton data key. |
+
+#### Return Values:
+
+| Name     | Type    | Description                      |
+| :------- | :------ | :------------------------------- |
+| `result` | bytes32 | Data key of `keyType` Singleton. |
+
+### generateArrayKey
+
+```solidity
+function generateArrayKey(
+    string memory keyName
+) internal pure returns (bytes32);
+```
+
+Generates a data key of `keyType` Array by hashing `keyName`.
+
+#### Parameters:
+
+| Name      | Type   | Description                                                            |
+| :-------- | :----- | :--------------------------------------------------------------------- |
+| `keyName` | string | The string that will be used to generate a data key of `keyType` Array |
+
+#### Return Values:
+
+| Name     | Type    | Description                  |
+| :------- | :------ | :--------------------------- |
+| `result` | bytes32 | Data key of `keyType` Array. |
+
+:::note
```

Comment:
I would suggest that we don't put the requirements as blockquote + not in a info callout.
To keep the layout consistent with the other Requirements pages. For instance in LSP0

<img width="801" alt="image" src="https://user-images.githubusercontent.com/31145285/197169976-cb6e1646-34db-4348-88a9-a5e1a355b935.png">

Reply:
Adding the info block was suggested by @YamenMerhi to have something similar to [authorizeOperator](http://docs.lukso.tech/standards/smart-contracts/lsp8-identifiable-digital-asset#authorizeoperator) in LSP8.